spa/src/feature-flags.ts

```diff
@@ -0,0 +1,3 @@
+const featureFlags = () => FRONTEND_FEATURE_FLAGS ?
+	JSON.parse(JSON.stringify(FRONTEND_FEATURE_FLAGS)) : null;
```

Comment:
Do we have lodash in spa? If so, then _.cloneDeep

Reply:
Whats the reason for cloning? 
The value for `FRONTEND_FEATURE_FLAGS` never changes.
Locally, its always gonna be `###FRONTEND_FEATURE_FLAGS###` and in production/staging, it will be set when the react app is built and never after, so don't really see the need for cloning it.